docs/CONTRIBUTING.md

```diff
@@ -2,60 +2,67 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 # Contributing
 
-- [Pull Requests](#pull-requests)
-  - [Commits](#commits)
 - [Requirements](#requirements)
-- [Development Workflow](#development-workflow)
-  - [Updating VS Code](#updating-vs-code)
-    - [Notes about Changes](#notes-about-changes)
-- [Build](#build)
-- [Testing](#testing)
-  - [Unit Tests](#unit-tests)
+- [Creating pull requests](#creating-pull-requests)
+  - [Commits and commit history](#commits-and-commit-history)
+- [Development workflow](#development-workflow)
+  - [Update VS Code](#update-vs-code)
+  - [Build](#build)
+  - [Test](#test)
+  - [Unit tests](#unit-tests)
   - [Integration Tests](#integration-tests)
-  - [End-to-End Tests](#end-to-end-tests)
+  - [End-to-end tests](#end-to-end-tests)
 - [Structure](#structure)
-  - [Modifications to VS Code](#modifications-to-vs-code)
+  - [Modifying VS Code](#modifying-vs-code)
   - [Currently Known Issues](#currently-known-issues)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 - [Detailed CI and build process docs](../ci)
 
-## Pull Requests
-
-Please create a [GitHub Issue](https://github.com/cdr/code-server/issues) to address any issue. You can skip this if the proposed fix is minor.
-
-In your Pull Requests (PR), link to the issue that the PR solves.
-
-Please ensure that the base of your PR is the **main** branch.
-
-### Commits
-
-We prefer a clean commit history. This means you should squash all fixups and fixup-type commits before asking for review (cleanup, squash, force-push). If you need help with this, feel free to leave a comment in your PR and we'll guide you.
-
 ## Requirements
 
-The prerequisites for contributing to code-server are almost the same as those for
-[VS Code](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#prerequisites).
-There are several differences, however. Here is what is needed:
+The prerequisites for contributing to code-server are almost the same as those
+for [VS
+Code](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#prerequisites).
```

Comment:
? Will this break the link being on two lines? Any reason is shouldn't be on one line? (too long?)

Reply:
I think the link should be fine, but the reason this sometimes happens is Product Docs enforces a strict line length so my IDE automatically does stuff like this across all MD files